secrethub/resource_dir.go

```diff
@@ -0,0 +1,117 @@
+package secrethub
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/secrethub/secrethub-go/internals/api"
+)
+
+func resourceDir() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDirCreate,
+		Read:   resourceDirRead,
+		Update: resourceDirUpdate,
+		Delete: resourceDirDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceDirImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"path": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The path of the directory.",
+			},
+			"force_destroy": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "A boolean that indicates all secrets and directories should be deleted from the directory so that it can be destroyed without error. These secrets and directories are not recoverable.",
```

Comment:
How about something like this:
```suggestion
				Description: "Whether to allow deleting this directory if it's not empty. When set to `false`, you'll get an error when trying to delete the directory if it still contains secrets.",
```

Reply:
I've added `directories` to the last bit: `if it still contains directories or secrets`